src/components/ShellView/ResultGraph.vue

```diff
@@ -122,6 +139,8 @@ export default {
     sidebarWidth: 500,
     graphWidth: 0,
     borderWidth: UI_SIZE.DEFAULT_BORDER_WIDTH,
+    hiddenNodes: [],
```

Comment:
You should not have these two arrays. G6 itself keeps track of the visibility state. https://antv-g6.gitee.io/en/docs/api/Items/itemMethods#itemisvisible